method/group/add-pobox.js

```diff
@@ -0,0 +1,75 @@
+const Crut = require('ssb-crut')
+const PoBox = require('../../spec/group/add-pobox')
+
+module.exports = function GroupPobox (ssb, keystore, state) {
+  const poBox = new Crut(ssb, PoBox)
+
+  function isValidRoot (content) {
+    isValidRoot.error = null
+    // fake a root message to validate static props!
+    const T = {
+      ...content,
+      tangles: {
+        poBox: {
+          root: null,
+          previous: null
+        }
+      }
+    }
+
+    if (!poBox.spec.isRoot(T)) {
+      isValidRoot.error = poBox.spec.isRoot.errorsString
+      return false
+    }
+
+    return true
+  }
+
+  function isValidUpdate (content) {
+    isValidUpdate.error = false
+    // fake an update message to validate mutable props only!
+    const T = { ...content }
+    delete T.tangles.group
+
+    // delete static props
+    delete T.version
+    delete T.root
+
+    if (!poBox.spec.isUpdate(T)) {
+      isValidUpdate.error = poBox.spec.isUpdate.errorsString
+      return false
+    }
+
+    return true
+  }
+
+  // TODO: choose whether to pass in the groupInit OR use the groupId to get it
```

Comment:
TODO: 

Would like your opinion on this @mixmix

Reply:
you have the keystore, so I think you can use the groupId to get that info
would certainly tidy this up

Reply:
const { root } = keystore.group.get(groupId)

root is groupInitMsgId!

OH, and :fire: you don't have a guarentee that you have this groupId in the keystore. So do this:

```js
const info = keystore.group.get(groupId)
if (!info) return cb(new Error('unknown groupId: ' + groupId)
```

Reply:
Added!